src/api.ts

```diff
@@ -11,6 +11,6 @@ function wait(delay: number) {
 export async function getPeople(): Promise<Person[]> {
   // keep this delay for testing purpose
   return wait(500)
-    .then(() => fetch(API_URL))
+    .then(() => fetch(`${API_URL}`))
```

Comment:
The template literal `${API_URL}` is unnecessary here. You can simply use `API_URL` directly in the fetch call.